recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - bluesky-darkframes
     - caproto
     - inflection
-    - nslsii >=0.0.17
+    - nslsii >=0.1.0
```

Comment:
This is for my benefit: why do you specify >=0.1.0? If that were not specified wouldn't the latest version be used?

Reply:
Just an extra paranoia in case some package decides to downgrade it.